### PR TITLE
Revert "e2e test for MachinePool and AWSManagedMachinePool"

### PR DESCRIPTION
**Pull request by ShaunakJoshi1407**
Reverts arlonproj/arlon#334
Reverting this change as the serviceaccount used for running the e2e tests doesn’t have the required permissions to create clusters with AWSMachinePool  and AWSManagedMachinePool components

### CODE REVIEW

File: pkg/basecluster/data.go [hunk 1]
```diff
@@ -67,8 +67,6 @@ nameReference:
   fieldSpecs:
   - path: spec/bootstrap/configRef/name
     kind: Machine
-  - path: spec/template/spec/bootstrap/configRef/name
```
Comment (bcle) [inline pkg/basecluster/data.go:70]: Why is this being removed? Was it incorrect?

File: pkg/basecluster/data.go [hunk 2]
```diff
@@ -155,10 +153,4 @@ nameReference:
   fieldSpecs:
   - path: spec/template/spec/infrastructureRef/name
     kind: MachinePool
-- kind: EKSConfig
```
Comment (bcle) [inline pkg/basecluster/data.go:158]: Why is this being removed? Was it incorrect?

**Review by bcle — COMMENTED** [review]
Some questions.

**Comment by Rohitrajak1807** [comment]
> Some questions.

We're undoing the PR. There's some issue with the AWS service account. It doesn't have the required permissions. CAPI team suggests to discuss this with Neha who'll be back on Friday. Until then this unblocks e2e test runs.

File: pkg/basecluster/data.go (hunk 1 above)
Comment (ShaunakJoshi1407) [reply pkg/basecluster/data.go:70]: Nope.....I will add it back here.....Just need to remove the template and make some changes in the setup script

**Pull request merged**